chapters/th/_toctree.yml

```diff
@@ -0,0 +1,4 @@
+- title: 3. การ fine-tune โมเดลที่ผ่านการเทรนมาแล้ว (pretrained model)
```

Comment:
Sorry for the hassle - since I merged #70 we now have a merge conflict between the `_toctree.yml` files. Would you like to resolve that or would you prefer that I do that?

Reply:
No problem, it was my mistake to begin with. Now I've fixed the conflict.

Reply:
Thanks!